### ISSUE TITLE
Fix accuracy of windows version

### ISSUE THREAD

**Issue by delebash** [jkirsons/FacialMotionCapture#4]
Any plans on trying to get as good as accuracy using dlib on linux as you do on windows?

Also this guy has a similar project but using open pose.  However there is no live version.  Just thought you might want to check it out.

https://github.com/nkeeline/OpenPose-to-Blender-Facial-Capture-Transfer

Thanks!

**Comment by Syllius24** [comment]
I would also love to see some refinement with the detection in Windows. Short of that, are there any suggestions for the actual video input? I've changed lighting and positioning, but the detection seems to be jittery regardless of the background / foreground clarity. I've been tempted to try makeup to see if it could help the tracking... The model appears to always be talking because of the jitters.

**Comment by delebash** [comment]
Yes I see that as well.  I ended up installing dlib on windows which helped .  Thank you for the program,  I will keep investigating :)

**Comment by Syllius24** [comment]
I'm trying to install Dlib. It was looking hopeful, only returning one error, until I installed CMAKE then my whole terminal lit up with red errors. I'm pretty sure now I'm facing version issues. Any link to a thorough walkthrough would be greatly appreciated. My head is dizzy jumping from too many tutorials today. lol

**Comment by delebash** [comment]
Sure.  I had problems with trying to install using the build tools as well.  What I ended up doing was uninstalling the build tools and then installing Visual Studio 2019, make sure you check Desktop Development with C++.  After that I ran pip install dlib and it compiled to version 19.09. My python version is 3.7.7.  Hope this works.

**Comment by delebash** [comment]
Just curious did you get dlib installed?  I also tried an earlier version of dlib install via pip and it would not compile.  So far 19.09 compiles fine.

**Comment by ptbw** [comment]
Installing dlib took some doing as its looks for Python.h in the system directory ie /usr/includes/PythonX.X even if you are using the built in Blender python. In the end I copied the contents of the system include directory into the Blender/Python/include directory. This allowed me to install dlib.
